lib/assembly-objectfile/content_metadata.rb

```diff
@@ -152,6 +155,14 @@ def self.create_content_metadata(params = {})
                 resource_type_description = resource_has_non_images ? resource_type_descriptions[:object] : resource_type_descriptions[:book]
               when :map
                 resource_type_description = resource_type_descriptions[:map]
+              when :'3d'
+                 resource_extensions = resource_files.collect {|obj| obj.ext}
+                 has_obj_files = resource_extensions.include?('.obj')
```

Comment:
per https://jirasul.stanford.edu/jira/browse/DEVQUEUE-229

any obj of type:
- obj
- ply
- threejs
- GLTF
and maybe others, should be covered here, I think.

Reply:
added these in latest commit